### PR TITLE
Create Github Action Verification

### PR DESCRIPTION
**Pull request by laeubi**
Having an automated verification build helps with contributing to this repository.

### CODE REVIEW

**Comment by laeubi** [comment]
@jbonofre can you take a look here also?

**Comment by laeubi** [comment]
Anyone? :-)

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: maven
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
+
+    # Optional: Uploads the full dependency graph to GitHub to improve the quality of Dependabot alerts this repository can receive
+    - name: Update dependency graph
+      uses: advanced-security/maven-dependency-submission-action@571e99aab1055c2e71a1e2309b9691de18d6b7d6
```
Comment (oscerd) [inline .github/workflows/maven.yml:35]: Is there a reason for pointing to that commit?

**Review by oscerd — COMMENTED** [review]
Just one question. Looks good to me

File: .github/workflows/maven.yml (hunk 1 above)
Comment (laeubi) [reply .github/workflows/maven.yml:35]: It was suggested by github at the time of my PR, maybe one need to update it in the meanwhile :-)

Comment (oscerd) [reply .github/workflows/maven.yml:35]: Let's merge

**Comment by jbonofre** [comment]
Thanks for the reminder and the merge @oscerd !
I forgot this one. LGTM.

**Pull request merged**